living_documentation_generator/github_projects.py

```diff
@@ -61,69 +61,55 @@ def send_graphql_query(self, query: str) -> dict[str, dict]:
         except Exception as e:
             logger.error("An error occurred: %s.", e, exc_info=True)
 
-        return {}
+        return
 
     def get_repository_projects(self, repository: Repository, projects_title_filter: list[str]) -> list[GithubProject]:
         projects = []
 
         # Fetch the project response from the GraphQL API
-        projects_from_repo_query = GithubProjectQueries.get_projects_from_repo_query(repository.owner.login,
-                                                                                     repository.name)
+        projects_from_repo_query = GithubProjectQueries.get_projects_from_repo_query(organization_name=repository.owner.login,
+                                                                                     repository_name=repository.name)
 
-        projects_from_repo_response = self.send_graphql_query(projects_from_repo_query)
+        projects_from_repo_response = self.__send_graphql_query(projects_from_repo_query)
+
+        if projects_from_repo_response is None:
+            logger.warning("Project response is None for repository `%s`.", repository.full_name)
```

Comment:
Idea: "Fetching GitHub project data - no project data for repository `%s`. No data received."

Reply:
Done in commit: